aot/src/test/java/com/dylibso/chicory/imports/SpecV1ElemHostFuncs.java

```diff
@@ -10,14 +10,30 @@
 import com.dylibso.chicory.runtime.HostTable;
 import com.dylibso.chicory.runtime.TableInstance;
 import com.dylibso.chicory.wasm.types.Limits;
+import com.dylibso.chicory.wasm.types.MutabilityType;
 import com.dylibso.chicory.wasm.types.Table;
 import com.dylibso.chicory.wasm.types.Value;
 import com.dylibso.chicory.wasm.types.ValueType;
 
 public class SpecV1ElemHostFuncs {
 
     public static HostImports fallback() {
-        return new HostImports();
+        return HostImports.builder()
+                .addGlobal(
```

Comment:
note to self: this API is better than before, but is still too verbose and detailed for 90% of the use cases.

Reply:
Agreed, i have some words to say about this in #441